client-app/src/examples/contact/mobile/details/DetailsPanel.ts

```diff
@@ -0,0 +1,145 @@
+import {XH, hoistCmp, uses} from '@xh/hoist/core';
+import {box, div, img, p, filler} from '@xh/hoist/cmp/layout';
+import {formField} from '@xh/hoist/mobile/cmp/form';
+import {form} from '@xh/hoist/cmp/form';
+import {panel} from '@xh/hoist/mobile/cmp/panel';
+import {select, textArea, textInput} from '@xh/hoist/mobile/cmp/input';
+import {button} from '@xh/hoist/mobile/cmp/button';
+import {toolbar} from '@xh/hoist/mobile/cmp/toolbar';
+import {Icon} from '@xh/hoist/icon';
+import {isEmpty, get} from 'lodash';
+
+import DetailsPanelModel from './DetailsPanelModel';
+import './DetailsPanelMobile.scss';
+
+export const detailsPanel = hoistCmp.factory({
+    model: uses(DetailsPanelModel),
+    render({model}) {
+        const {currentRecord, visible} = model;
+
+        return panel({
+            className: `tb-mobile-contact-details-panel${visible ? '--details-visible' : ''}`,
```

Comment:
We should use a mobile navigator to handle drilldowns. This will allow us to:
+ Make the details its own page, meaning you don't need to handle visibility / transitions
+ Route to specific contacts by name
+ Support gesture navigation on device, etc